### ISSUE TITLE
Wrong path removing files from file storage

### ISSUE THREAD

**Issue by parera10** [cshum/imagor#472]
Docker image version: 1.4.13

My configuration:
```
  FILE_RESULT_STORAGE_BASE_DIR: /data/results
  FILE_RESULT_STORAGE_EXPIRATION: 48h
  FILE_RESULT_STORAGE_EXPIRATION_SECONDS: "172800"
  FILE_STORAGE_BASE_DIR: /data/storage
  FILE_STORAGE_EXPIRATION: 48h
  FILE_STORAGE_EXPIRATION_SECONDS: "172800"
  IMAGOR_RESULT_STORAGE_PATH_STYLE: digest
  IMAGOR_STORAGE_PATH_STYLE: digest
  IMAGOR_UNSAFE: "1"
  PORT: "8000"
  PROMETHEUS_BIND: :7000
  PROMETHEUS_PATH: /metrics
```

Ignore "*_STORAGE_EXPIRATION_SECONDS" vars. They are used for a cleaning storage cronjob.

So imagor is using hashed paths for saving in file storage but it's trying to remove the files using the origin encoded url in the path and obviously no file there. Some logs:

```
2024-08-23 10:02:12.669Z imagor {"level":"warn","ts":1724407332.6698532,"caller":"imagor/imagor.go:586","msg":"delete","key":"https://s3.amazonaws.com/bucketname/hotel/0a80b01d-beed-4341-b99c-0e5bd57c2dae","error":"remove /data/storage/https%3A/s3.amazonaws.com/bucketname/hotel/0a80b01d-beed-4341-b99c-0e5bd57c2dae: no such file or directory"}
```

**Comment by cshum** [comment]
https://github.com/cshum/imagor/pkgs/container/imagor/262909827?tag=master

You may test the docker build at master branch:

```
ghcr.io/cshum/imagor@sha256:039522d05b9e0bcd45e7749833dec36fc3988e2ec9d24515612a6cb547ee0634
```

**Comment by cshum** [comment]
Available in v1.4.15
